### PR TITLE
New Org Mode Cheat Sheet

### PR DESCRIPTION
**Pull request by erickgnavar**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New Org mode Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Add org-mode cheatsheet

###### Which issues (if any) does this fix?

None

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/org_mode_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@erickgnavar Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/org-mode.json [hunk 1]
```diff
@@ -0,0 +1,518 @@
+{
+    "id": "org_mode_cheat_sheet",
+    "name": "Org mode",
+    "description": "Org mode cheatsheet",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/org-mode.json]: Les make a description  which tells about the contents of the cheat sheet. Refer other cheat sheets for example

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=org-mode cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/org_mode_cheat_sheet_mobile.png?8KsPPr91LY)

File: share/goodie/cheat_sheets/json/org-mode.json [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+    "aliases": [
+        "org-mode", "orgmode"
+    ],
+
+    "template_type": "keyboard",
+
+    "section_order": [
+        "TODO Items and Checkboxes",
+        "Timestamps",
+        "Clocking time",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/org-mode.json]: Lets make this "Clocking Time"
Dont forget to update the same below

File: share/goodie/cheat_sheets/json/org-mode.json (hunk 1 above)
Comment (tagawa) [inline share/goodie/cheat_sheets/json/org-mode.json]: @erickgnavar I updated the description in the IA page so feel free to copy and paste that: https://duck.co/ia/view/org_mode_cheat_sheet

**Comment by erickgnavar** [comment]
I update the commit with the suggestions @jophab @tagawa

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=org-mode cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/org_mode_cheat_sheet_mobile.png?W9rfDOXwTw)

**Comment by tagawa** [comment]
Hi @erickgnavar. Thank you for this cheat sheet and the updates, and sorry for the delay in reviewing it.

As you can see, it's now on the [beta server for testing](https://beta.duckduckgo.com/?q=org-mode%20cheatsheet) so we can let the community try it out for a few days. If there are no more comments after that, then we should be good to merge.

**Comment by tagawa** [comment]
No further review comments so let's merge this! Thank you @erickgnavar for creating this, and @jophab for the feedback. It should go live in a few days.

And if you'd like to continue contributing, we're currently on a programming mission, aiming to improve Instant Answers for developers. More info on how to help out here: https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53

**Comment by daxtheduck** [comment]
@erickgnavar this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by tagawa** [comment]
@erickgnavar I'm pleased to say this is live now - thank you!
https://duckduckgo.com/?q=org-mode+cheatsheet&ia=cheatsheet

**Pull request merged**